corehq/apps/integration/payments/services.py

```diff
@@ -167,3 +169,20 @@ def _payment_is_verified(case_data: dict):
 
 def _payment_already_submitted(case_data: dict):
     return case_data.get(PaymentProperties.PAYMENT_SUBMITTED) == 'True'
+
+
+def get_payment_batch_numbers_for_domain(domain):
+    case_ids = (
+        CaseSearchES()
+        .domain(domain)
+        .case_type(MOMO_PAYMENT_CASE_TYPE)
+        .values_list("_id", flat=True)
+    )
+
+    batch_numbers = set()
+    for case_id_chunk in chunked(case_ids, CHUNK_SIZE):
```

Comment:
I'd suggest using a much higher chunk size for this.
Looks like it's currently 100, which seems reasonable to requesting payments, where this constant is used other than this.

Can we push it to 1k? You could check the chunk size used at other places for ideas.